composer.json

```diff
@@ -44,7 +44,7 @@
         "drupal/config_split": "^2.0@RC",
         "drupal/core-composer-scaffold": "^9.1",
         "drupal/core-project-message": "^9.1",
-        "drupal/core-recommended": "^9.1",
+        "drupal/core-recommended": "^9.1@stable",
```

Comment:
If you do a `ddev composer update "drupal/core-*" --with-all-dependencies` you'll get D9.5.9 and this line will remain the same. I don't quite know how different the `composer.lock` file will be when you do so. But the upgrade process for Drupal if using `drupal/core-recommended` is `composer update "drupal/core-*" --with-all-dependencies`. I always defer to this page: https://www.drupal.org/docs/updating-drupal/updating-drupal-core-via-composer

Reply:
I did one earlier! :-)

Reply:
In the end, the differences were trivial. Typical.
<img width="832" alt="Cursor_and_git_diff_--cached" src="https://user-images.githubusercontent.com/158008/235971468-584b2a5d-8b04-4942-ad15-3f246640b80c.png">